samples/xamarin-forms/Cinephile/Cinephile.Core/Infrastructure/DynamicDataExtensions.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using Cinephile.Core.Models;
+using DynamicData;
+using DynamicData.Kernel;
+
+namespace Cinephile.Core.Infrastructure
+{
+    public static class DynamicDataExtensions
+    {
+        public static void EditDiff<TDomainEntity, TKey>(this SourceCache<TDomainEntity, TKey> sourceCache, IEnumerable<TDomainEntity> items, int? offset = null, int pageSize = 25) where TDomainEntity : Movie
```

Comment:
Worth making this line due to having almost 200 chars... It makes it more readable. Probably something like this code below?

```csharp
        public static void EditDiff<TDomainEntity, TKey>(this SourceCache<TDomainEntity, TKey> sourceCache,
            IEnumerable<TDomainEntity> items,
            int? offset = null,
            int pageSize = 25)
            where TDomainEntity : Movie
```